src/Microsoft.Security.Utilities.Core/Microsoft.Security.Utilities.Core.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <IsPackable>true</IsPackable>
-    <TargetFrameworks>net462;netstandard2.0;net6.0;net8.0</TargetFrameworks>
```

Comment:
>net462 [](http://example.com/codeflow?start=22&length=6)

in theps1 there is
`$Configuration\net462\SecurityUtilitiesApiUtilizationExample.exe`
do we need to keep and add instead of replace?

Reply:
@shaopeng-gh Yea that just adds validation for the compatibility.